edx_name_affirmation/handlers.py

```diff
@@ -35,56 +41,52 @@ def verified_name_approved(sender, instance, **kwargs):  # pylint: disable=unuse
         )
 
 
-def idv_attempt_handler(attempt_id, user_id, status, photo_id_name, full_name, **kwargs):
```

Comment:
This is only ever triggered by a `SoftwareSecurePhotoVerification` and can removed since we will no longer be creating those records.

_Why remove this?_
We run `idv_update_verified_name_task` on an attempt id which is not unique between the two data models making it somewhat ambiguous as to what field that id is for. While we could start passing two fields (type, id) throughout the service that's going to create a mess we'll need to come back around and remove.

Reply:
to close out this thread. We've decided to move forward with this as is. Given the following:

1. name-affirmation is only used by 2U
2. name-affirmation can drop support for SoftwareSecurePhotoVerifications prior to it's full removal in the LMS
3. functionally this would work as it did before for any verifications using the new models